functions/src/lib/id.ts

```diff
@@ -0,0 +1,13 @@
+import {compose} from 'ramda';
+import {v5 as uuidv5} from 'uuid';
+import bs58 from 'bs58';
+
+import config from './config';
+
+const {DAILY_ID_SALT} = config;
+
+const uuidv5Bytes = (name: string) => uuidv5(name, DAILY_ID_SALT, []);
+const createId = compose(bs58.encode, uuidv5Bytes);
+
+export const createRctUserId = (roomId: string, userId: string) =>
+  createId(`room:${roomId}:user:${userId}`);
```

Comment:
Nit pick, but what is `rct`? Wouldn't `daily` make more sense?

Reply:
Haha, yes, reused the mighty Örjans naming convention. Make kind of sense, they migrated video platform and maybe we will too 😄

Reply:
rct, `real time communication` apparently